Metalama.Framework.Engine/SyntaxGeneration/ContextualSyntaxGenerator.cs

```diff
@@ -829,7 +829,7 @@ public SyntaxList<TypeParameterConstraintClauseSyntax> TypeParameterConstraintCl
 
     public CastExpressionSyntax SafeCastExpression( TypeSyntax type, ExpressionSyntax syntax )
     {
-        if ( syntax is CastExpressionSyntax cast && cast.Type.IsEquivalentTo( type, topLevel: false ) )
+        if ( syntax is CastExpressionSyntax cast && cast.Type.IsEquivalentTo( type, false ) )
```

Comment:
I generally like having named arguments when it's not clear what the argument means. Why are they being removed here?